docs/en/setup/backend/backend-storage.md

```diff
@@ -49,6 +49,12 @@ Since 8.8.0, SkyWalking rebuilds the ElasticSearch client on top of ElasticSearc
 correct request formats according to the server-side version, hence you don't need to download different binaries
 and don't need to configure different storage selectors for different ElasticSearch server-side versions anymore.
 
+Since 9.2.0, SkyWalking support merge all metrics/meter indices into one physical index `metrics-all`.
+Provide config `SW_STORAGE_ES_LOGIC_SHARDING` to sharding metrics indices into multi physical indices as previous version, but disabled by default. 
+If disabled, all metrics will persist in one physical index, to reduce the number of physical indices.
+If enabled, the metrics will persist in multi physical indices according to their model index name and aggregation function name, 
+the metrics which has the same aggregation function name will persist in one physical index.
```

Comment:
```suggestion
Since 9.2.0, SkyWalking merges all metrics/meter indices into one physical index template `metrics-all` on the default setting.
Provide system environment variable(`SW_STORAGE_ES_LOGIC_SHARDING`) to shard metrics indices into multi-physical indices as the previous versions(one index template per metric/meter aggregation function).
In the current one index mode, users still could choose to adjust ElasticSearch's shard number(`SW_STORAGE_ES_INDEX_SHARDS_NUMBER`) to scale out.
```